src/cargo/util/errors.rs

```diff
@@ -1,3 +1,4 @@
+extern crate curl_sys;
```

Comment:
I moved it to src/cargo/lib.rs I think thats the "root". Let me know if that is wrong.